plugins/dependencies/src/main/kotlin/dependencies/Http4k.kt

```diff
@@ -4,103 +4,106 @@ private val GROUP = "org.http4k"
 
 object Http4k : DependencyGroup(
     group = GROUP,
-    usePlatformConstraints = true,
+    usePlatformConstraints = false,
     rawRule = """
         org.http4k:http4k-*
             ^^^^^^
     """.trimIndent()
 ) {
-    val bom = "org.http4k:http4k-bom:_"
-
-    val aws = module("http4k-aws")
-    val cloudnative = module("http4k-cloudnative")
-    val contract = module("http4k-contract")
-    val core = module("http4k-core")
-    val graphql = module("http4k-graphql")
-    val incubator = module("http4k-incubator")
-    val jsonrpc = module("http4k-jsonrpc")
-    val metricsMicrometer = module("http4k-metrics-micrometer")
-    val multipart = module("http4k-multipart")
-    val opentelemetry = module("http4k-opentelemetry")
-    val realtimeCore = module("http4k-realtime-core")
-    val resilience4j = module("http4k-resilience4j")
-    val securityOauth = module("http4k-security-oauth")
+    val bom: String get() {
+        usePlatformConstraints = true
```

Comment:
Nice.

Reply:
Now that I'm thinking about it again…
I think it won't be that easy: in this form, one module could affect another, which can lead to surprising results that depend on non deterministic execution order.

We would want to attach the BoM activation on a per configuration, or a per module basis. Not sure how we can do that at the moment though.